src/ui/public/doc_table/components/table_row/details.html

```diff
@@ -15,6 +15,13 @@
     >
       View single document
     </a>
+    <a 
```

Comment:
This link isn't keyboard accessible, since it doesn't have an `href` element. Since this is an action rather than a navigation to a different page, it should be a `button` (and just styled whatever way you wish): https://github.com/elastic/kibana/blob/master/style_guides/accessibility_guide.md#use-button-and-a-href

Reply:
Done.